### PR TITLE
added multiple environment example

### PR DESCRIPTION
**Pull request by mreider**
I thought our public repo deserved an explanation of sending logs to multiple environments. This is working for me, but I was careful to state the following to make sure people know this has not been thoroughly tested. 

_The following example serves as a potential reference implementation. Results may vary._

### CODE REVIEW

File: example/config-map-multi.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluentd-ingest-configuration
+  namespace: dynatrace
+data:
+  CLUSTER_ID: "b51a38b1-example-436a-9c0a-b0fb214a0fa7"
```
Comment (meibensteiner) [inline example/config-map-multi.yaml:7]: `CLUSTER_ID: "${YOUR_CLUSTER_ID}"`

File: example/fluentd-multi.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      initContainers:
+      - name: fluentd-permission-fix
+        image: busybox
+        command: ["sh","-c","touch /var/log/fluentd.pos; chmod -R 770 /var/log/fluentd.pos; "]
+        volumeMounts:
+        - name: var-log
+          mountPath: /var/log/
+      containers:
+      - name: fluentd
+        image: mreider/fluentd:2
```
Comment (meibensteiner) [inline example/fluentd-multi.yaml:33]: They should really build their own image, I think
`${YOUR_FLUENTD_IMAGE}`

File: example/config-map-multi.yaml (hunk 1 above)
Comment (mreider) [reply example/config-map-multi.yaml:7]: sounds good.

File: example/fluentd-multi.yaml (hunk 2 above)
Comment (mreider) [reply example/fluentd-multi.yaml:33]: that's fine.

**Review by mreider — COMMENTED** [review]
LGTM

**Pull request closed without merging**